### PR TITLE
St svetlana seda lesson2 level1

### PR DESCRIPTION
**Pull request by java0711**
Не понимаю или добавила правильноБ не могу найти класс у себя. Проверка

### CODE REVIEW

File: src/main/java/students/student_svetlana_seda/lesson_2/level_1/Task1.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package students.student_svetlana_seda.lesson_2.level_1;
+
+import java.util.Scanner;
+
+public class Task1 {
+    public static void main(String[] args) {
+
+        Scanner scanner=new Scanner(System.in);
```
Comment (antdolg1) [inline src/main/java/students/student_svetlana_seda/lesson_2/level_1/Task1.java:8]: Please add spaces before and after assignment operator (here and in other places).

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Please add spaces where required.

File: src/main/java/students/student_svetlana_seda/lesson_2/level_1/Task1.java (hunk 1 above)
Comment (java0711) [reply src/main/java/students/student_svetlana_seda/lesson_2/level_1/Task1.java:8]: Sorry, I did not understand, space between line 8 and 9?

Comment (java0711) [reply src/main/java/students/student_svetlana_seda/lesson_2/level_1/Task1.java:8]: or: 
Scanner scanner = new Scanner(System.in);

Comment (antdolg1) [reply src/main/java/students/student_svetlana_seda/lesson_2/level_1/Task1.java]: `Scanner scanner=new Scanner(System.in);` -> `Scanner scanner = new Scanner(System.in);`

**Review by antdolg1 — APPROVED** [review]

**Pull request merged**